error/src/lib.rs

```diff
@@ -500,6 +501,7 @@ impl CaliptraError {
     pub const FW_PROC_MAILBOX_PROCESS_FAILURE: CaliptraError = CaliptraError::new_const(0x01020007);
     pub const FW_PROC_MAILBOX_STASH_MEASUREMENT_MAX_LIMIT: CaliptraError =
         CaliptraError::new_const(0x01020008);
+    pub const FW_PROC_MAILBOX_RESERVED_PAUSER: CaliptraError = CaliptraError::new_const(0x01020009);
```

Comment:
We have the potential for some divergence on these error codes if we add them in each branch at the same time. Are we okay with the same error potentially having a slightly different code on 1.x vs 2.0? Or do we need to ensure they are consistent?

Reply:
As I've mentioned before, branching many of these crates is a mistake.

Reply:
@nquarton We'll can do a sweep on main-2.x at a later point for the error codes and ensure they are in sync with main.